More API client examples in the dev documentation (#2002)
google/timesketch: docs/developers/api-client.md

+ A pandas DataFrame if the `as_pandas=True` is set
+ A python dict (default behavior)

## Add data

#### Manually add events to the sketch

> Fill in your event data Put in your message, date and timestamp description

Reviewer: ```suggestion
Fill in your event data, you need at least these fields: message, date and timestamp description
```

```python

message = "foobar"
date = "2020-08-06T12:48:06.994188Z"
timestamp_desc = "Test_description"

> #attributes: A dict of extra attributes to add to the event.  

Reviewer: ```suggestion
# Attributes: A dict of extra attributes to add to the event.  
```

attributes = {"a": "alpha", "o": "omega", "g": "gamma"}

> #tags: A list of strings to include as tags.

Reviewer: ```suggestion
# Tags: A list of strings to include as tags.
```
